src/api/app/models/user.rb

```diff
@@ -374,7 +374,11 @@ def render_axml(watchlist = false)
   end
 
   def home_project_name
-    "home:#{login}"
+    if login.starts_with?('.', ':', '_')
+      "home:project#{login}"
```

Comment:
If anyone knows a better, valid char (@ doesn't work) or a better solution in general, please speak up:smiley:

Reply:
I wouldn't waste a lot of thought on this - as long it's possible at all. But a : in the middle of the project name is a problem in itself - as as `home:project:bgeuken` would be subproject of `home:project` - just as a login of `bgeuken:mudder` would result in `home:bgeuken:mudder` and you got access in it.

As such using `home:user_#{id}` if login is not resulting in a valid project name might be an alternative